frontend/src/components/order/DetailedOrder.vue

```diff
@@ -29,19 +56,25 @@ export default {
     await this.getOrderItems();
   },
   methods: {
-    getOrderItems: function() {
+    getOrderItems: async function() {
       if (this.order.order_id === "") {
         return;
       }
-      const path = `http://localhost:5000/orders/${this.order.order_id}/items`;
-      axios
-        .get(path)
-        .then(response => {
-          this.order_items = response.data;
-        })
-        .catch(error => {
-          console.log(error);
-        });
+      const response = await getOrderItems(this.order.order_id);
+      this.order_items = response.data;
+      await this.getBeers();
+    },
+    getBeers: async function() {
+      if (this.order.order_id === "") {
+        return;
+      }
+
+      for (let i=0; i<this.order_items.length; i++) {
+        const response = await getBeer(this.order_items[i].beer_id);
+        this.order_items[i]['beer'] = response.data;
+        this.order_items[i]['item_id'] += 1; //help
+        this.order_items[i]['item_id'] -= 1; //help
```

Comment:
Comment est-ce que je peux faire ça autrement pour le forcer à updater la component?

Reply:
En faisant `this.order_items = this.update` ça le fait pas ?

Reply:
Non, le key est sur item_id et il ne change pas vu que je fais juste rajouter une clé à l'objet